test/js/length-value.js

```diff
@@ -1,15 +1,15 @@
 suite('LengthValue', function() {
-  test('fromValue returns instanceof a LengthValue and StyleValue', function() {
-    var fromVal = (new LengthValue()).fromValue(9.2, 'px');
-    assert.instanceOf(fromVal, SimpleLength, 'A new fromValue should be an instance of SimpleLength');
-    assert.instanceOf(fromVal, LengthValue, 'A new fromValue should be an instance of LengthValue');
-    assert.instanceOf(fromVal, StyleValue, 'A new fromValue should be an instance of StyleValue');
+  test('fromDictionary returns instanceof a LengthValue and StyleValue', function() {
+    var fromDictionary = (new LengthValue()).fromValue(9.2, 'px');
+    assert.instanceOf(fromDictionary, SimpleLength, 'A new fromDictionary should be an instance of SimpleLength');
+    assert.instanceOf(fromDictionary, LengthValue, 'A new fromDictionary should be an instance of LengthValue');
+    assert.instanceOf(fromDictionary, StyleValue, 'A new fromDictionary should be an instance of StyleValue');
   });
 
   test('fromDictionary returns instanceof a LengthValue and StyleValue', function() {
-    var fromDict = (new LengthValue()).fromDictionary({px: 10});
-    assert.instanceOf(fromDict, CalcLength, 'A new fromDictionary should be an instance of CalcLength');
-    assert.instanceOf(fromDict, LengthValue, 'A new fromDictionary should be an instance of LengthValue');
-    assert.instanceOf(fromDict, StyleValue, 'A new fromDictionary should be an instance of StyleValue');
+    var fromDictionary = (new LengthValue()).fromDictionary({px: 10});
```

Comment:
Both above comments also apply here, except you can call this on calcLength or value.